src/content/codelabs/index.md

```diff
@@ -245,12 +260,6 @@ Learn how to use Flutter with other technologies.
   can interact with the text interface or stream a voice
   interaction via the built-in microphone of a mobile device.
 
-* [Introduction to Flame with Flutter][]<br>
```

Comment:
Why is this being removed? It's still on codelabs.developers.google.com (I hope)

Reply:
It's still there. I just moved it to the new Games section.